layers/meta-opentrons/recipes-robot/opentrons-usb-bridge/files/opentrons-usb-bridge.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Opentrons USB Bridge
+Requires=nginx.service
+After=nginx.service
+
+[Service]
+Type=notify
```

Comment:
talked offline, but unless you're using python systemd bindings to call `notify()`, you probably want `simple` here